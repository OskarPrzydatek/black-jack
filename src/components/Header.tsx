import * as S from '../styles/Header.style';

interface HeaderProps {
  title: string;
}

export default function Header({ title }: HeaderProps) {
  return <S.Header>{title}</S.Header>;
}
