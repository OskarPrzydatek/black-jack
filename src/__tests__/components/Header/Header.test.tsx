import renderer from 'react-test-renderer';

import Header from '../../../components/Header';

describe('Header', () => {
  test('component snapshot', () => {
    const tree = renderer.create(<Header title="Header" />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});

export {};
