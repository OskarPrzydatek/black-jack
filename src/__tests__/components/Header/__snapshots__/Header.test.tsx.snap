// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header component snapshot 1`] = `
<h1
  className="sc-bcXHqe ghEhJc"
>
  Header
</h1>
`;
